### ISSUE TITLE
노션 api 명세 대신 asciidoc 사용 제안

### ISSUE THREAD

**Issue by mingd1023** [PoolC/Mincho#84]
## 🗣 설명

프론트 짜다 보니 노션에 정리된 api 명세대로 하기 까다롭다는 생각이 들었습니다.ㅜㅜ
낯설더라도 asciidoc 이용하는건 어떨까요?
지금까지 완성된 api 명세 모두 asciidoc으로 옮겨주시고, 새로 완성되는 api 명세도 asciidoc 이용하시면 좋을 것 같아요
asciidoc 작성이 포함된 PR 날릴 때마다 저를 PR에서 언급 (@mingd1023 이런식으로!)해주시고 PR 본문에 asciidoc 업데이트 된 내용 적어주시면,
제가 pull 받고 알아서 asciidoc 돌려보겠습니다. 

## 📋 체크리스트

> 구현해야하는 이슈 체크리스트

1. 
2.

**Comment by hcpak** [comment]
네 알겠습니다. comment-api 완료하고, 현재까지 발생한 에러 수정한 다음에 asciidoc 작성하도록 하겠습니다.
